moler/config/__init__.py

```diff
@@ -85,21 +87,18 @@ def load_config(config=None, from_env_var=None, config_type='yaml'):
         add_devices_only = True
         loaded_config.append(config)
 
-    if (config_type != 'dict') and (config_type != 'yaml'):  # no other format supported yet
-        raise WrongUsage("Unsupported config_type: '{}'. Allowed are: 'dict' or 'yaml'.".format(config_type))
+    if config is None and from_env_var is None:
+        raise WrongUsage("Provide either 'config' or 'from_env_var' parameter (none given).")
+    elif not from_env_var and (not isinstance(config, six.string_types) and not isinstance(config, dict)):  # no other format supported yet
+        raise WrongUsage("Unsupported config type: '{}'. Allowed are: 'dict' or 'yaml'.".format(type(config)))
```

Comment:
`Allowed are: 'dict' or 'yaml'` suggests that you can provide config as yaml string. Confusing.
change to:
`Allowed are: 'dict' or 'str' holding config filename (file is in YAML format)`

Reply:
5f07205bb653e034a28e4baa13c691d023b1f200